eng/push-tarball.sh

```diff
@@ -129,6 +129,16 @@ pushd "$vmr_path"
     git push -u upstream "$new_branch_name"
 
     echo "Creating PR from $new_branch_name to $target_branch"
-    curl -H 'Content-Type: application/json' -d "$data" "$pr_url"
+    curlResult=$(curl -H 'Content-Type: application/json' -d "$data" "$pr_url" 2>/dev/null)
```

Comment:
I believe we use `snake_case` in bas (and in the rest of this file) for variable names

Reply:
Since we are redirecting STDERR to /dev/null and are not checking the exit code, how do we learn about an error? IS there reason to discard the STDERR?

Reply:
I had assumed that just the request response would be sufficient, but now I am including the errors as well. Thank you :)